### PR TITLE
Added support for function return statements in opcode and Expr struct.

### PR DESCRIPTION
**Pull request by nmdis1999**
Changes:
1.  Updated `ukiyo.l` and 'ukiyo.y` files to have `return` keyword and production rule.
2. Added `return` variant to `Expr` in `config_ast.rs`.
3. Added `tests/files/return_stmt.ukiyo` to check if opcode is working as intended.

### CODE REVIEW

**Comment by ltratt** [comment]
@nmdis1999 You want to add someone as an Assignee (not as a Reviewer). I suggest picking Lukas.

File: src/lib/vm.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ impl fmt::Display for Types {
 fn vm(
     prog: Vec<OpCode>,
     locals: &mut Vec<Types>,
-    functions: &mut Vec<Function>,
+    functions: &mut Vec<Function>
```
Comment (ptersilie) [inline src/lib/vm.rs:42]: Looks like your editor has reformatted your code again. Buildbot is not gonna let that through. Maybe try having a quick look if you can teach your editor to either use the same formatting as `cargo fmt` or turn off formatting in the editor entirely.

**Comment by ptersilie** [comment]
Looks good apart from one small formatting comment.

**Comment by ptersilie** [comment]
Sorry, I forgot to mention: we never force push to PRs unless given the go ahead by the reviewer. But you couldn't have known that. Typically the author would push a normal commit (or merge commit if pulling in other changes) and then when the reviewer is happy they'll ask for the commits to be squashed. Only then can the PR author force push.

**Pull request merged**